packages/@jxa/sdef-to-dts/bin/cmd.js

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env node
+
+const fs = require("fs");
+const path = require("path");
+const child_process = require("child_process");
+const transform = require("../lib/sdef-to-dts").transform;
+
+function usage() {
+  console.log();
+  console.log("Usage: npx @jxa/sdef-to-dts inFile [out]");
+  console.log();
+  console.log("  Converts Script Definitions into TypeScript Type Definitions")
+  console.log();
+  console.log("  inFile - path to an Application.app to read");
+  console.log("  out - path to an Application.d.ts or a directory to write to");
+  console.log();
+}
+
+if (process.argv.length !== 3 && process.argv.length !== 4) {
```

Comment:
I like to use command parser library like [meow](https://github.com/sindresorhus/meow#readme) prefer than hand written.

[meow](https://github.com/sindresorhus/meow#readme) includes following feature

- Show usage(`--help` by `autoHelp` option)
- Correct exit status
- Parse flags like `--out [outdir]`

Reply:
```
Usage: npx @jxa/sdef-to-dts <input> --output <output>
```

In [meow](https://github.com/sindresorhus/meow#readme)
```js
const cli = meow(`
	Usage
	  $ npx @jxa/sdef-to-dts <input> --output <output>
          
          <input> - path to an Application.app to read
	Options
	  --output, -o  path to an Application.d.ts or a directory to write to

	Examples
	  $ npx @jxa/sdef-to-dts /Applications/Safari.app --output ./safari.d.ts
`, {
	flags: {
		output: {
			type: 'string',
			alias: 'o'
		}
	},
        autoVersion: true, // <= --version
        autoHelp: true // <= --help 
});

// ... 
```


- `cli.input[0]` is `inFile`
- `cli.flags.output` is `[outDir or outPath]`